pkg/schema/tracker.go

```diff
@@ -329,3 +340,208 @@ func (tr *Tracker) CreateTableIfNotExists(table *filter.Table, ti *model.TableIn
 func (tr *Tracker) GetSystemVar(name string) (string, bool) {
 	return tr.se.GetSessionVars().GetSystemVar(name)
 }
+
+// GetDownStreamIndexInfo gets downstream PK/UK(not null) Index.
+// note. this function will init downstreamTrack's table info.
+func (tr *Tracker) GetDownStreamIndexInfo(tableID string, originTi *model.TableInfo, tctx *tcontext.Context, task string, tidbConn *conn.BaseConn) (*model.IndexInfo, error) {
+	if tr.downstreamTrack == nil {
+		tr.downstreamTrack = make(map[string]*ast.CreateTableStmt)
+	}
+	createTableStmt := tr.downstreamTrack[tableID]
+	if createTableStmt == nil {
+
+		log.L().Info(fmt.Sprintf("DownStream schema tracker init: %s", tableID))
+
+		rows, err := tidbConn.QuerySQL(tctx, fmt.Sprintf("SHOW CREATE TABLE %s", tableID))
```

Comment:
I think using `tidbConn.QueryRowSQL` is much easier here.
If not, you have to run `rows.Close` after the if condition of `err`.

Reply:
ok, i will try.